modules/lib/lib-schema/src/main/resources/lib/xp/examples/schema/updateXData.js

```diff
@@ -0,0 +1,61 @@
+var schemaLib = require('/lib/xp/schema');
+var assert = require('/lib/xp/testing');
+
+/* global log*/
+
+let resource = `<?xml version="1.0" encoding="UTF-8"?>
+        <x-data xmlns="urn:enonic:xp:model:1.0">
+          <display-name>Virtual X-data</display-name>
+          <description>X-data description</description>
+          <form>
+            <input type="TextLine" name="label">
+              <label>Label</label>
+              <immutable>false</immutable>
+              <indexed>true</indexed>
+              <custom-text/>
+              <help-text/>
+              <occurrences minimum="0" maximum="2"/>
+            </input>
+          </form>
+        </x-data>`
+
+// BEGIN
+// Update virtual mixin.
+var result = schemaLib.updateSchema({
+    name: 'myapp:mydata',
+    type: 'XDATA',
+    resource: resource
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=9736743)
```suggestion
    resource
```